nasa-gibs/worldview
# WV-3060: Update DSWx version

## Description

Fixes #WV-3060 .

Update OPERA Surface Water Extent to new version. (Removes  "_provisional" from layer name)

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?v=-49.13327397339445,16.682817527358374,58.2838458066835,64.41743802284853&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,OPERA_L3_Dynamic_Surface_Water_Extent-HLS(disabled=5),VIIRS_NOAA21_CorrectedReflectance_TrueColor(hidden),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=true&t=2024-06-28-T04%3A00%3A00Z)
2. Ensure the Dynamic Surface Water Extent layer loads, displays imagery, Available Imagery Dates works as expected (by querying CMR), you can take an image snapshot, make sure Surface Water Extent tour story works.



## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
